src/lib/dto/v5/swing.dto.v5.ts

```diff
@@ -1,6 +1,6 @@
 import { AbstractSensorEventResponseV5 } from './abstract-sensor-event.dto.v5';
-import { SwingMetricDtoV5 } from './swing-metric.dto.v5';
 import { CollectionResponseV5 } from './collection.dto.v5';
+import { SwingMetricDtoV5 } from './swing-metric.dto.v5';
```

Comment:
Should probably keep the naming consistent with `PitchingMetricDTOV5` (capital DTO).

Reply:
For sure!